modules/comments/api/getPollComments.ts

```diff
@@ -51,6 +51,7 @@ export async function getPollComments(
     return {
       comment,
       isValid,
+      completed: i === 0 ? false : transaction && transaction.confirmations > 10,
```

Comment:
Awesome, looks really good. Curious about this line though, why is `i === 0 ? false`?